tests/test_django_log_formatter.py

```diff
@@ -193,21 +193,13 @@ def test_does_not_log_personally_identifiable_information_when_log_sensitive_use
         overrides = {
             "user": self._create_user(),
         }
-
+        
```

Comment:
We need some kind of pre-commit hook for these whitespaces.

Reply:
Resolved in https://github.com/uktrade/django-log-formatter-asim/pull/4/commits/173ac5c08f4bad3875414f0a48844c166da78bc9.